test/e2e_node/summary_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/onsi/gomega/types"
 )
 
-const restartCount = 3
+const restartCount = 1
```

Comment:
Is there any way in this test to make sure the container do exit at least? My concern is without such check, we are testing what we expect to test.